sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -386,26 +387,39 @@ export class AssessmentControllerService {
       applicationId,
       options?.studentId,
     );
-    return requestAssessmentSummary.concat(appeals);
+    const applicationOfferingChangeRequests =
+      await this.getApplicationOfferingChangeRequestsByStatus(
+        applicationId,
+        [
+          ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
+          ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+          ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+        ],
+        options?.studentId,
+      );
+    return requestAssessmentSummary
+      .concat(appeals)
+      .concat(applicationOfferingChangeRequests);
```

Comment:
How will be the order for the pending and denied assessment list on the ministry side?

Reply:
@ann-aot do you mean similar to what we do for the Students below?
https://github.com/bcgov/SIMS/blob/debecde07be2cd15da230c8d33bebebb036e4f11/sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts#L164

Reply:
@andrewsignori-aot, I think we were planning to do like pending first (all trigger types order by date) and denied (all trigger types order by date) for the ministry as we had before. Let me know if Iam missing anything.

Reply:
I do not see a problem either way @ann-aot, I do not recall which was the agreement and if there was a requirement anywhere about it. Maybe we can add a comment near to return of this method to make it clear.

Reply:
The code reference mentioned by @andrewsignori-aot, at that time I believe the decision was to show the latest submitted request at the top and then go chronologically backwards from there. Not sure if there was a change in requirement after that.